### PR TITLE
Implement ccapi with fabric gateway

### PR DESCRIPTION
**Pull request by mikaellafs**
- Routes that communicate with fabric gateway start with `/gateway/`

### CODE REVIEW

**Review by andremacedopv — APPROVED** [review]
LGTM

File: fabric/network.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ fi
 export FABRIC_CFG_PATH=${PWD}/configtx
 export VERBOSE=false
 # export COMPOSE_PROJECT_NAME=net
-export IMAGE_TAG=2.2.3
+export IMAGE_TAG=2.4.7
```
Comment (samuelvenzi) [inline fabric/network.sh:21]: How about we move this to `v2.5.3`? 👀

File: ccapi/handlers/invokeGateway.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+	if err != nil {
+		common.Abort(c, http.StatusBadRequest, err)
+		return
+	}
+
+	txName := c.Param("txname")
+
+	// Get endorsers names
+	var endorsers []string
+	endorsersQuery := c.Query("@endorsers")
```
Comment (samuelvenzi) [inline ccapi/handlers/invokeGateway.go:43]: @mikaellafs What's the `@endorsers`? Can you explain it or provide a reference link for me?

**Review by samuelvenzi — APPROVED** [review]
Great job! I think my only input is to use `v2.5.x`. A next step is update `startDev.sh` with the latest fabric-samples' `network.sh` since we are still creating a system channel.

@andremacedopv @mikaellafs

File: ccapi/handlers/invokeGateway.go (hunk 2 above)
Comment (mikaellafs) [reply ccapi/handlers/invokeGateway.go:43]: You can explicitly select the organizations to endorse a transaction. Check this [Targeting specific endorsement peers](https://hyperledger-fabric.readthedocs.io/en/latest/gateway.html#targeting-specific-endorsement-peers)

File: fabric/network.sh (hunk 1 above)
Comment (mikaellafs) [reply fabric/network.sh:21]: Alright! :ok_hand:

**Review by andremacedopv — APPROVED** [review]

**Review by samuelvenzi — APPROVED** [review]

**Pull request merged**